sympy/printing/fcode.py

```diff
@@ -365,6 +365,12 @@ def _print_Float(self, expr):
             return "%sd%s" % (printed[:e], printed[e + 1:])
         return "%sd0" % printed
 
+    def _print_Relational(self, expr):
+        lhs_code = self._print(expr.lhs)
+        rhs_code = self._print(expr.rhs)
+        op = expr.rel_op
+        return "{0} {1} {2}".format(lhs_code, op, rhs_code)
```

Comment:
For fortran case, I think that it can be better to merge `_print_Equality` and `_print_Unequality` here.

Reply:
I have removed `_print_Equality` and `_print_Unequality`. The tests already exist under `test_fcode`  as `test_fcode_Relational()`.
Please review.